src/sandstorm/spk.c++

```diff
@@ -944,6 +948,15 @@ private:
   kj::MainBuilder::Validity doPack() {
     ensurePackageDefParsed();
 
+    if (!packageDef.getManifest().getAppTitle().hasDefaultText()) {
+      context.exitError(kj::str("Expected manifest.appTitle.defaultText to be nonempty."));
```

Comment:
This error message feels like it may take more cognitive power than necessary to understand. Maybe a better one would be:

    Your package manifest is missing `appTitle`. In `sandstorm-pkgdef.capnp`, under `pkgdef.manifest`, add something like:
        appTitle = (defaultText = "My App"),